agent/wsclient/client.go

```diff
@@ -33,16 +33,16 @@ import (
 	"sync"
 	"time"
 
+	"crypto/tls"
```

Comment:
Can you move `crypto/tls` into the block at the top with all the other stdlib imports?